pkg/gatherers/clusterconfig/mutatingwebhookconfigurations.go

```diff
@@ -14,17 +14,30 @@ import (
 	"github.com/openshift/insights-operator/pkg/utils/anonymize"
 )
 
-// GatherMutatingWebhookConfigurations collects MutatingWebhookConfiguration resources
-// Relevant OpenShift API docs:
-//   - https://docs.openshift.com/container-platform/4.8/rest_api/extension_apis/mutatingwebhookconfiguration-admissionregistration-k8s-io-v1.html
+// GatherMutatingWebhookConfigurations Collects MutatingWebhookConfiguration resources.
 //
-// * Location in archive: config/mutatingwebhookconfigurations
-// * Id in config: clusterconfig/mutating_webhook_configurations
-// * Since versions:
-//   - 4.7.40+
-//   - 4.8.24+
-//   - 4.9.11+
-//   - 4.10+
+// ### API Reference
+// - https://docs.openshift.com/container-platform/4.8/rest_api/extension_apis/mutatingwebhookconfiguration-admissionregistration-k8s-io-v1.html
+//
+// ### Sample data
+// - docs/insights-archive-sample/config/mutatingwebhookconfigurations
+//
+// ### Location in archive
+// - `config/mutatingwebhookconfigurations/{resource}.json`
+//
+// ### Config ID
+// `clusterconfig/mutating_webhook_configurations`
+//
+// ### Released version
+// - 4.10.13
```

Comment:
Again wrong released version.

Reply:
Fixing. It was `4.10.3` [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=2027629) [RN](https://access.redhat.com/errata/RHSA-2022:0056)